libdistopia/src/distopia.cpp

```diff
@@ -869,21 +871,39 @@ namespace distopia {
         return HWY_DYNAMIC_DISPATCH(CalcDihedralsTriclinicDouble)(a, b, c, d, n, box, out);
     }
     HWY_DLLEXPORT template <> void CalcDistanceArrayNoBox(const double *a, const double *b, int na, int nb, double *out) {
+        if (nb < 4) {
```

Comment:
Not sure how much it matters for an array this small, but the minimum size is an  `__mm128d` for `<double>` of width 2 rather than 4, but don't think it really matters either way.